avr/cores/megacommand/MCL/A4SeqTrack.h

```diff
@@ -7,12 +7,8 @@
 class A4SeqTrack : public ExtSeqTrack {
 
 public:
-  A4SeqTrack(Encoder *e1 = NULL,
-               Encoder *e2 = NULL, Encoder *e3 = NULL, Encoder *e4 = NULL)
-      : ExtSeqTrack(e1, e2, e3, e4) {}
-  bool handleEvent(gui_event_t *event);
-  void display();
-  void setup();
+  A4SeqTrack()
```

Comment:
hmm why did we put the encoders here?

Reply:
No idea. I think the history here is messed up. Showing very old changes.

Reply:
`ext_conv` was quite a while ago